dummy/validation/mealFieldValidation.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable require-jsdoc */
+
+export default class Validator {
+  /**
+     *
+     * @param {*} req
+     * @param {*} res
+     * @param {*} next
+     * @param {*} json
+     * @param {*} message
+     */
+
+  // VALIDATE MEAL FORM FIELD
+  static mealValidation(req, res, next) {
+    const { img, title, description, price } = req.body;
+    try {
+      const regex = /^[a-zA-Z\s]*$/;
+      if (img === '') {
```

Comment:
You want to use `img.trim() === ''` here. With the way it is, if the img is `'         '`, it will pass and be added to the DB. Using trim removes all whitespaces